src/libDirectoryService/DirectoryService.h

```diff
@@ -378,6 +375,10 @@ class DirectoryService : public Executable, public Broadcastable {
   /// committee.
   std::atomic<Mode> m_mode;
 
+  std::mutex m_mutexAllPOW;
```

Comment:
Why was this moved?

Reply:
Move from private to public, to allow mediator access it to calculate shard node number